.flake8

```diff
@@ -1,5 +1,5 @@
 [flake8]
-max-line-length = 80
+max-line-length = 100
```

Comment:
With `B950` we allow line length up to 10% more without raising an error. Personally I find 110 characters a bit long for one line (doesn't fit my screen).

Reply:
I'd rather have 100 as a hard limit and not let even a 10% more. The choice of 100 is because is the default of black which seems logical (less than that starts looking like Fortran...)

Reply:
https://github.com/psf/black#line-length
Blacks default is 88. That would be `B950` with `max-line-length = 80` (that is where I got it from). I don't mind hard limit at 100 would also work.

Reply:
I guess black is able to see the .pylintrc file I have in my $HOME. I've believed since I started using it that the default was 100, my life is a lie :___